csharp/test/Drivers/BigQuery/DriverTests.cs

```diff
@@ -18,9 +18,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Apache.Arrow.Adbc.Drivers.BigQuery;
 using Apache.Arrow.Adbc.Tests.Metadata;
 using Apache.Arrow.Adbc.Tests.Xunit;
 using Apache.Arrow.Ipc;
+using Apache.Arrow.Types;
+using Google.Apis.Auth.OAuth2;
+using Google.Cloud.BigQuery.V2;
+using Moq;
```

Comment:
Given last summer's controversy, I'm reluctant to add Moq as a new dependency. How hard would it be to avoid?